src/helpers/helpers.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable valid-jsdoc */
+import validator from 'validator';
+
+const helpers = {
+  /**
+   * @description This method helps validate passwords.
+   * @param  {string} password The password you are trying to validate
+   * @retxurns {object} Contains valid (bool) and invalidMessages (array[strings])
+   */
+  validPassword: password => {
+    let valid = true;
+    const invalidMessages = [];
```

Comment:
I like the implementation of your validation @olorunwalawrence , but could you please reach out tomorrow for more information on how we agreed to implement validation using joi?